app/controllers/application_controller.rb

```diff
@@ -2165,10 +2165,10 @@ def visibility_box_edit
 
   def get_record_display_name(record)
```

Comment:
@Yadnyawalkya this is a common method that is called from other controllers as well, so we should not change the behaviour of this method, that can cause an unexpected behaviour on other screens. I suggest changing
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/miq_ae_class_controller.rb#L2566 to
```
record_name = element.name
```
let me know if that works for you or if you have questions.